plugin.video.rivedila7/resources/lib/plugin.py

```diff
@@ -62,6 +62,7 @@ def add_directory_item_nodup(parameters, li, title='', folder=True, is_live=Fals
 
 
 def play_dirette(url, live):
+    xbmc.log('PLAY DIRETTE', xbmc.LOGINFO)
```

Comment:
why loginfo and not logdebug?

Reply:
It simple, you di not need enable debug 🤟

Reply:
That sounds awesome but:

https://kodi.wiki/view/Add-on_rules

addons should use the xbmc.log() method to write to the logfile and use the debug logging level only. print statements should not be used.

Reply:
Bureaucrat 😅